quickstart.md

```diff
@@ -380,618 +380,173 @@ $ ssh -i ccv0-ssh root@$(kubectl get service ccv0-ssh -o jsonpath="{.spec.cluste
-
-3. Publish the encrypted image to your registry.
-
-#### Deploy encrypted image
-
-1. Build rootfs with EAA component:
-
-Specify `AA_KBC=eaa_kbc` parameters when using kata-containers `rootfs.sh` scripts to create rootfs.
-
-2. Launch Verdictd
-
-Verdictd performs remote attestation at runtime and provides the key needed to decrypt the image.
-It is actually both Key Broker Service and Attestation Service of EAA.
-So when deploy the encrypted image, Verdictd is needed to be launched:
-
-```sh
-verdictd --listen <$ip>:<$port> --mutual
-```
-
-> **Note** The communication between Verdictd and EAA KBC is based on rats-tls,
-so you need to confirm that [rats-tls](https://github.com/inclavare-containers/rats-tls) has been correctly installed in your running environment.
+Now you can use CoCo Key Broker System cluster to create encrypted container image and deploying it, refer to: [Deploy and Configure tenant-side Key Broker System cluster](#deploy-and-configure-tenant-side-coco-key-broker-system-cluster).
+You also can use [EAA Verdictd](guides/eaa-verdictd-guide.md) to do that, refer to [Verdictd guide](guides/eaa-verdictd-guide.md).
 
-3. Agent Configuration
-
-Add configuration `aa_kbc_params= 'eaa_kbc::<$IP>:<$PORT>'` to agent config file, the IP and PORT should be consistent with verdictd.
-
-### Use offline SEV KBC and simple-kbs (SEV)
-
-#### Create an Encrypted Image
-
-If SSH access to the container is desired, create a keypair:
-
-```
-ssh-keygen -t ed25519 -f encrypted-image-tests -P "" -C "" <<< y
-```
-
-The above command will save the keypair in a file named `encrypted-image-tests`.
+### SEV
+The `kata-qemu-sev` runtime class must be specified.
 
-Here is a sample Dockerfile to create a docker image:
+Now you can use [simple-kbs](guides/sev-guide.md) to create encrypted container image and depolying it on SEV.
+Please refer to [sev-guide.md](guides/sev-guide.md)
 
-```
-FROM alpine:3.16
+### Deploy and Configure tenant-side CoCo Key Broker System cluster
 
-# Update and install openssh-server
-RUN apk update && apk upgrade && apk add openssh-server
+A tenant-side CoCo Key Broker System cluster include:
+- Key Broker Service (KBS): Brokering service for confidential resources.
+- Attestation Service (AS): Verifier for remote attestation.
+- Reference Value Provicer Service (RVPS): Provides reference values for AS.
+- CoCo Keyprovider: Component to encrypt the images following ocicrypt spec.
 
-# Generate container ssh key
-RUN ssh-keygen -t ed25519 -f /etc/ssh/ssh_host_ed25519_key -P ""
+To quick start the KBS cluster, a `docker-compose` yaml is provided to launch.
 
-# A password needs to be set for login to work. An empty password is
-# unproblematic as password-based login to root is not allowed.
-RUN passwd -d root
+```shell
+# Clone KBS git repository
+git clone https://github.com/confidential-containers/kbs.git
+cd kbs
```

Comment:
Or we directly give the link to the `docker-compoer` here, as we do not the source code of kbs to deploy docker-compose
```suggestion
wget https://github.com/confidential-containers/kbs/blob/main/docker-compose.yml
```